<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentManager</class>
 <widget class="QWidget" name="ContentManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="fileView"/>
   </item>
   <item>
    <widget class="QCheckBox" name="useLinks">
     <property name="text">
      <string>Использовать ссылки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoGroup">
     <property name="text">
      <string>Выбирать группу автоматически</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
